frontend/src/views/tests/organisationManagement.test.js

```diff
@@ -95,17 +95,15 @@ describe('Create Organization', () => {
   });
 
   it('should enable create organization button when the value is changed', async () => {
-    const { createButton } = setup();
+    const { user, createButton } = setup();
     const nameInput = screen.getAllByRole('textbox')[0];
-    const user = userEvent.setup();
     await user.type(nameInput, 'New Organization Name');
     expect(createButton).toBeEnabled();
   });
 
   it('should navigate to the newly created organisation detail page on creation success', async () => {
-    const { router, createButton } = setup();
+    const { user, router, createButton } = setup();
     const nameInput = screen.getAllByRole('textbox')[0];
-    const user = userEvent.setup();
     await user.type(nameInput, 'New Organization Name');
     const subscriptionType = screen.getByRole('combobox');
     fireEvent.mouseDown(subscriptionType);
```

Comment:
Were there any elements that affected your decision to leave this `fireEvent` alone? Isn't it true that `await user.click(subscriptionType);` would also work?

Reply:
Are you talking about the `fireEvent.mouseDown(subscriptionType);`? If so, the reason why I didn't touch it is I hadn't debugged the code to see if we were specifically testing against an extended mouse click.

Reply:
It was not particularly tested against the mouse click event. I had trouble triggering the display content by clicking the dropdown button in the test case, so I resorted to using the `mouseDown` event. Surprisingly, it now appears to be functioning, although I'm not sure what's causing it to be so.